earthpy/tests/test_spatial.py

```diff
@@ -290,4 +303,51 @@ def test_crop_image_fails_empty_list(basic_image_tif, basic_geometry):
         es.crop_image(list(), basic_geometry)
     with rio.open(basic_image_tif) as src:
         with pytest.raises(ValueError):
+
             es.crop_image(src, list())
+
+
+def test_hillshade_shape(hillshade_arr):
+    """If provided with input array not of shape (rows, columns),
+    ValueError returned."""
+
+    # Test data
+    arr = hillshade_arr
+
+    # Check ValueError
+    with pytest.raises(
+        ValueError, match="Input array should be of shape: rows, columns"
```

Comment:
let's revisit this error message given my previous comments.

Reply:
Modified to match ValueError("Input array should be two-dimensional")